editor/code/syntaxes/coq.json

```diff
@@ -0,0 +1,229 @@
+{
+    "fileTypes": ["v"],
+    "name": "Coq",
+    "patterns": [
+        {
+            "match": "\\b(From|Require|Import|Export|Local|Global|Include)\\b",
+            "comment": "Vernacular import keywords",
```

Comment:
Technically `Global` and `Local` are import modifiers and not "Import Keywords" as `Require` or `Import` are; for example you have `Global Variable` etc...